openapi.yaml

```diff
@@ -9259,3 +9341,149 @@ components:
+        * `dep-jira-issues-count` - number of JIRA issues mapped to deployed pull requests.
+        * `dep-jira-bug-fixes-count` - number of deployed bug fixes.
+      enum:
+      - dep-count
+      - dep-duration
+      - dep-success-count
+      - dep-failure-count
+      - dep-success-ratio
+      - dep-size-prs
+      - dep-size-releases
+      - dep-size-lines
+      - dep-size-commits
+      - dep-prs-count
+      - dep-releases-count
+      - dep-lines-count
+      - dep-commits-count
+      - dep-repository-frequency
+      - dep-jira-issues-count
+      - dep-jira-bug-fixes-count
+      type: string
+    DeploymentMetricsRequest:
+      additionalProperties: false
+      description: Request body of `/metrics/deployments`, the deployments selector.
+      properties:
+        for:
+          description: List of repository groups for which to calculate the metrics.
+          items:
+            $ref: '#/components/schemas/RepositorySet'
+          type: array
+        with:
+          description: List of developer groups for which to calculate the metrics.
+          items:
+            $ref: '#/components/schemas/DeploymentWith'
+          type: array
+        metrics:
+          description: Requested metric identifiers.
+          items:
+            $ref: '#/components/schemas/DeploymentMetricID'
+          type: array
+        pr_labels_include:
+          description: At least one of these labels must be present in at least one
+            deployed PR.
+          items:
+            type: string
+          type: array
+        pr_labels_exclude:
+          description: None of these labels must be present in each deployed PR.
+          items:
+            type: string
+          type: array
+        jira:
+          $ref: '#/components/schemas/JIRAFilter'
+        with_labels:
+          description: Deployments should contain at least one of the specified label
+            values. `null` matches any label value and effectively checks the label
+            existence.
+          type: object
+        without_labels:
+          description: Deployments may not contain the specified label values. `null`
+            matches any label value and effectively ensures that the label does not
+            exist.
+          type: object
+        environments:
+          description: List of environment groups for which to calculate the metrics.
+          items:
+            items:
+              type: string
+            type: array
+          type: array
+        date_from:
+          description: Date from when to start measuring the metrics.
+          format: date
+          type: string
+        date_to:
+          description: Date up to which to measure the metrics.
+          format: date
+          type: string
+        timezone:
+          $ref: '#/components/schemas/TimeZone'
+        granularities:
+          items:
+            $ref: '#/components/schemas/Granularity'
+          type: array
+        quantiles:
```

Comment:
If I summarize, quantiles are applied only to:

- `dep-duration`
- `dep-size-*`
- `release-time-to-deploy`

Reply:
All the aggregations, naturally.